frappe/public/js/frappe/form/controls/attach.js

```diff
@@ -79,10 +79,12 @@ frappe.ui.form.ControlAttach = frappe.ui.form.ControlData.extend({
 		this.value = value;
 		if(this.value) {
 			this.$input.toggle(false);
-			if(this.value.indexOf(",")!==-1) {
-				var parts = this.value.split(",");
-				var filename = parts[0];
-				dataurl = parts[1];
+			// value can also be using this format: FILENAME,DATA_URL
+			// Important: We have to be careful because normal filenames may also contain ","
+			let two_part_matches = this.value.match(/^([^:]+),(.+):(.+)$/);
+			if (two_part_matches !== null) {
+				let filename = two_part_matches[1];
```

Comment:
use `var filename` instead of `let filename`, `let` is limited to the block in which it is declared while variable declared with `var` has the global scope. This is also why Slider is failing.